### DISCUSSION TITLE
Is there some faring method for a whole mesh?

### DISCUSSION THREAD

**Discussion by fgbxx** [cnr-isti-vclab/meshlab#1460] [Show and tell]
Hi, dear all,
   I want to smooth a mesh and keep the volume as possible , but laplace smooth method will make mesh shrink, but we can see in Geomagic software, the function QuickFair can do whole mesh fair and change volume very little. So Is there some filter to do similar fair for whole mesh ?

**Comment by jmespadero** [comment]
"Taubin smooth" is the filter you are looking for.

I literally would change the "Laplacian smooth" filter description to include the phrase "You probably are more interested in Taubin Smooth than in this filter"

**Reply by fgbxx** [reply]
Thank you for your advice. I tried the method , the parameters are hard to be tuned. 
Is there some whole mesh fair method for it?

**Reply by jmespadero** [reply]
Yes, It's hard to tune Lambda an Mu parameters without changing the volume. Try to just adjust the `Smoothing Steps` number to a value between 1 and 50.
